Core/EventHandlers.lua

```diff
@@ -1987,6 +1987,20 @@ function R:OnLootReady(event, ...)
 			end
 		end
 
+		--Handling for Winter Night Dreamsaber mount
+		if Rarity.isOpening and Rarity.lastNode == L["Dreamseed Cache"] then
+			Rarity:Debug("Detected Opening on " .. Rarity.lastNode .. " (method = SPECIAL)")
```

Comment:
If this entire block was a separate function (e.g., `OnDreamseedCacheOpened`) it could be tested more easily.

Reply:
it only needs to be a pretty basic function also right?

`function Rarity:OnDreamseedCacheOpened()
	addAttemptForItem(dreamseedMounts, "mounts")
end`

assuming dreamseedMounts is a table of the mounts and that works within the function

Reply:
I guess? Depends on how complicated the logic has to be for this and the other mounts. (also, may need one call per item)

Reply:
for loop seems to work, but since we now know it drops from only the epic seed its abit irrelevant unless we cannot track it that way